### PR TITLE
Update Tournaments page

### PR DESCRIPTION
**Pull request by fajar13k**
### Description
- Removes Beautify in Table
- Replace old forum link to newer one
- Change date formatting to `YYYY-MM-DD`
- Add TWC 2018 and MWC 7K 2018 to Official Tournaments
- Add past installments of community tournaments (SOFT, AUS/NZ, SST, etc.)
- Add recent community tournaments (SOFT, CFC)

### Status
Pending Review / Finished

### Notes
Don't merge yet, currently trying to find someone to update `de.md`

### CODE REVIEW

File: wiki/Tournaments/en.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Official tournaments hosted by the Tournament Management.
 osu! World Cup is a country-based 4v4 Team VS tournament following double-elimination rule. It is an international event and is usually done yearly with the osu!communities around the world working together to make a tournament to create a competitive nature.
 
 | Name | Start | End | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") |
-|------|-------|-----|----------------------------------------------------|------------------------------------------------------|------------------------------------------------------|
+| ------ | ------- | ----- |---------------------------------------------------- | ------------------------------------------------------ | ------------------------------------------------------ |
```
Comment (Joehuu) [inline wiki/Tournaments/en.md]: Use only three characters per column. Do for every instance.
i.e.
`| --- | --- | --- | --- | --- | --- |`

File: wiki/Tournaments/en.md [hunk 2]
```diff
@@ -26,19 +26,20 @@ osu! World Cup is a country-based 4v4 Team VS tournament following double-elimin
 ### ![](/wiki/shared/mode/taiko.png) Taiko World Cup
```
Comment (Joehuu) [inline wiki/Tournaments/en.md]: `Taiko World Cup` -> `osu!taiko World Cup`

File: wiki/Tournaments/en.md [hunk 3]
```diff
@@ -26,19 +26,20 @@ osu! World Cup is a country-based 4v4 Team VS tournament following double-elimin
...
-| Taiko World Cup \#3                                | 2012-09-22 | 2013-03-24 | ![][flag_TW] Taiwan A | ![][flag_HK] Hong Kong A | ![][flag_JP] Japan A |
+| Taiko World Cup \#3 | 2012-09-22 | 2013-03-24 | ![][flag_TW] Taiwan A | ![][flag_HK] Hong Kong A | ![][flag_JP] Japan A |
 | [Taiko World Cup 2014](/wiki/Tournaments/TWC/2014) | 2014-01-16 | 2014-03-09 | ![][flag_JP] Japan | ![][flag_HK] Hong Kong | ![][flag_FR] France |
 | [Taiko World Cup 2015](/wiki/Tournaments/TWC/2015) | 2015-01-12 | 2015-03-15 | ![][flag_JP] Japan | ![][flag_TW] Taiwan | ![][flag_FR] France |
 | [Taiko World Cup 2016](/wiki/Tournaments/TWC/2016) | 2016-02-09 | 2016-04-24 | ![][flag_JP] Japan | ![][flag_TW] Taiwan | ![][flag_CA] Canada |
 | [Taiko World Cup 2017](/wiki/Tournaments/TWC/2017) | 2017-02-07 | 2017-04-16 | ![][flag_JP] Japan | ![][flag_TW] Taiwan | ![][flag_HK] Hong Kong |
+| [Taiko World Cup 2018](/wiki/Tournaments/TWC/2018) | 2018-04-07 | 2017-05-13 | ![][flag_JP] Japan | ![][flag_TW] Taiwan | ![][flag_CA] Canada |
 
 ### ![](/wiki/shared/mode/catch.png) Catch the Beat World Cup
```
Comment (Joehuu) [inline wiki/Tournaments/en.md]: `Catch the Beat World Cup` -> `osu!catch World Cup`

File: wiki/Tournaments/en.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ Official tournaments hosted by the Tournament Management.
 osu! World Cup is a country-based 4v4 Team VS tournament following double-elimination rule. It is an international event and is usually done yearly with the osu!communities around the world working together to make a tournament to create a competitive nature.
```
Comment (Joehuu) [inline wiki/Tournaments/en.md]: `osu! World Cup` -> `The osu! World Cup`

`osu!communities` -> `osu! communities`

File: wiki/Tournaments/en.md [hunk 5]
```diff
@@ -26,19 +26,20 @@ osu! World Cup is a country-based 4v4 Team VS tournament following double-elimin
...
 | [Taiko World Cup 2012](/wiki/Tournaments/TWC/2012) | 2012-02-03 | 2012-04-01 | ![][flag_JP] Japan A | ![][flag_HK] Hong Kong A | ![][flag_JP] Japan B |
-| Taiko World Cup \#3                                | 2012-09-22 | 2013-03-24 | ![][flag_TW] Taiwan A | ![][flag_HK] Hong Kong A | ![][flag_JP] Japan A |
+| Taiko World Cup \#3 | 2012-09-22 | 2013-03-24 | ![][flag_TW] Taiwan A | ![][flag_HK] Hong Kong A | ![][flag_JP] Japan A |
 | [Taiko World Cup 2014](/wiki/Tournaments/TWC/2014) | 2014-01-16 | 2014-03-09 | ![][flag_JP] Japan | ![][flag_HK] Hong Kong | ![][flag_FR] France |
 | [Taiko World Cup 2015](/wiki/Tournaments/TWC/2015) | 2015-01-12 | 2015-03-15 | ![][flag_JP] Japan | ![][flag_TW] Taiwan | ![][flag_FR] France |
 | [Taiko World Cup 2016](/wiki/Tournaments/TWC/2016) | 2016-02-09 | 2016-04-24 | ![][flag_JP] Japan | ![][flag_TW] Taiwan | ![][flag_CA] Canada |
 | [Taiko World Cup 2017](/wiki/Tournaments/TWC/2017) | 2017-02-07 | 2017-04-16 | ![][flag_JP] Japan | ![][flag_TW] Taiwan | ![][flag_HK] Hong Kong |
+| [Taiko World Cup 2018](/wiki/Tournaments/TWC/2018) | 2018-04-07 | 2017-05-13 | ![][flag_JP] Japan | ![][flag_TW] Taiwan | ![][flag_CA] Canada |
```
Comment (Joehuu) [inline wiki/Tournaments/en.md]: `Taiko World Cup 2018` -> `osu!taiko World Cup 2018`

Idk about the others(old ones). Probably check their pages logos and change accordingly.

File: wiki/Tournaments/en.md [hunk 6]
```diff
@@ -26,19 +26,20 @@ osu! World Cup is a country-based 4v4 Team VS tournament following double-elimin
...
 
 ### ![](/wiki/shared/mode/catch.png) Catch the Beat World Cup
 
 | Name | Start | End | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") |
-|------|-------|-----|----------------------------------------------------|------------------------------------------------------|------------------------------------------------------|
+| ------ | ------- | ----- | ---------------------------------------------------- | ------------------------------------------------------ | ------------------------------------------------------ |
 | Catch the beat World Cup 2012 | 2011-11-05 | 2012-05-02 | ![][flag_CN] China | ![][flag_TW] Taiwan | ![][flag_CL] Chile |
 | Catch the beat World Cup 2013 | 2013-01-15 | 2013-03-10 | ![][flag_CL] Chile | ![][flag_TW] Taiwan | ![][flag_AR] Argentinia/Colombia |
```
Comment (Joehuu) [inline wiki/Tournaments/en.md]: This line and before.
`beat` -> `Beat`

File: wiki/Tournaments/en.md [hunk 7]
```diff
@@ -13,7 +13,7 @@ Official tournaments hosted by the Tournament Management.
...
 
 | Name | Start | End | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") |
-|------|-------|-----|----------------------------------------------------|------------------------------------------------------|------------------------------------------------------|
+| ------ | ------- | ----- |---------------------------------------------------- | ------------------------------------------------------ | ------------------------------------------------------ |
 | [osu! World Cup #1](/wiki/Tournaments/OWC/1) | 2011-01-31 | 2011-04-08 | ![][flag_TW] Taiwan | ![][flag_NL] Netherlands | ![][flag_KR] South Korea |
 | [osu! World Cup #2](/wiki/Tournaments/OWC/2) | 2011-10-28 | 2012-03-24 | ![][flag_KR] South Korea | ![][flag_JP] Japan | ![][flag_BR] Brazil |
 | [osu! World Cup #3](/wiki/Tournaments/OWC/3) | 2012-10-25 | 2013-01-20 | ![][flag_KR] South Korea | ![][flag_JP] Japan | ![][flag_TW] Taiwan |
```
Comment (Joehuu) [inline wiki/Tournaments/en.md]: Remove the link. Page doesn't exist.

File: wiki/Tournaments/en.md [hunk 8]
```diff
@@ -26,19 +26,20 @@ osu! World Cup is a country-based 4v4 Team VS tournament following double-elimin
 ### ![](/wiki/shared/mode/taiko.png) Taiko World Cup
 
 | Name | Start | End | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") |
-|------|-------|-----|----------------------------------------------------|------------------------------------------------------|------------------------------------------------------|
+| ------ | ------- | ----- | ---------------------------------------------------- | ------------------------------------------------------ | ------------------------------------------------------ |
 | [Taiko World Cup 2011](/wiki/Tournaments/TWC/2011) | 2011-03-12 | 2011-04-29 | ![][flag_TW] Taiwan | ![][flag_JP] Japan | ![][flag_CN] China |
 | [Taiko World Cup 2012](/wiki/Tournaments/TWC/2012) | 2012-02-03 | 2012-04-01 | ![][flag_JP] Japan A | ![][flag_HK] Hong Kong A | ![][flag_JP] Japan B |
```
Comment (Joehuu) [inline wiki/Tournaments/en.md]: Remove the link. Page doesn't exist.

File: wiki/Tournaments/en.md (hunk 5 above)
Comment (fajar13k) [reply wiki/Tournaments/en.md]: Apparently every year it has different titles, will follow the articles.

File: wiki/Tournaments/en.md [hunk 9]
```diff
@@ -1,7 +1,7 @@
 Tournaments
 ===============
 
-Section dedicated for Tournaments and/or Competitions. [Forum link here.](https://osu.ppy.sh/forum/55)
+Section dedicated for Tournaments and/or Competitions. [Forum link here.](https://osu.ppy.sh/community/forums/55)
 
 Official World Cups
```
Comment (TPGPL) [inline wiki/Tournaments/en.md]: just `Official` should be enough i think

**Comment by Joehuu** [comment]
@fajar13k why is there translation diffs for `pl`,`pt-br`, and `zh`? Also it should have outdated labels.

**Comment by fajar13k** [comment]
@Joehuu Those files are like that before i made the PR, so i reverted it like that too. Also, some translations are left behind too far. Besides that, I will label them as outdated now.

**Comment by Joehuu** [comment]
Alright, the diffs were changing some weird spaces, lines, etc.

**Comment by TPGPL** [comment]
Superseded by #1525

**Pull request closed without merging**